docker-compose.yml

```diff
@@ -72,6 +72,10 @@ x-sccache: &sccache
   SCCACHE_REGION:
   SCCACHE_S3_KEY_PREFIX: ${SCCACHE_S3_KEY_PREFIX:-sccache}
 
+x-cpp: &cpp
+  ARROW_RUNTIME_SIMD_LEVEL:
+  ARROW_SIMD_LEVEL:
+
```

Comment:
Also would like your opinion here @kou

Reply:
I don't object `cpp` but I like `simd`.

Reply:
I like `cpp` better as we may want to add other inherited variables here.